hikey.mk

```diff
@@ -372,12 +372,35 @@ atf-fb-clean:
 ################################################################################
 # l-loader
 ################################################################################
-lloader: arm-tf
-	$(MAKE) -C $(LLOADER_PATH) BL1=$(ARM_TF_PATH)/build/hikey/$(ARM_TF_BUILD)/bl1.bin CROSS_COMPILE="$(CCACHE)$(AARCH32_CROSS_COMPILE)" PTABLE_LST=linux-$(CFG_FLASH_SIZE)g
+.PHONY: lloader-bin
+lloader-bin: arm-tf atf-fb
+	cd $(LLOADER_PATH) && \
+		ln -sf $(ARM_TF_PATH)/build/hikey/$(ARM_TF_BUILD)/bl1.bin && \
+		ln -sf $(ATF_FB_PATH)/build/hikey/$(ATF_FB_BUILD)/bl1.bin fastboot.bin && \
+		$(AARCH32_CROSS_COMPILE)gcc -c -o start.o start.S && \
+		$(AARCH32_CROSS_COMPILE)ld -Bstatic -Tl-loader.lds -Ttext 0xf9800800 start.o -o loader && \
+		$(AARCH32_CROSS_COMPILE)objcopy -O binary loader temp && \
+		python gen_loader_hikey.py -o l-loader.bin --img_loader=temp --img_bl1=bl1.bin --img_ns_bl1u=fastboot.bin
```

Comment:
That is seriously ugly and fragile, too.
Can't you fix the lloader Makefile instead and keep the previous code here?

Reply:
Ok. I'll submit a PR to the l-loader repo and make the necessary changes here once it's merged.

Reply:
Sounds good!

Reply:
@jforissier l-loader Makefile PR has been merged and instructions here updated. Please ack before I rebase, squash and tag.

Reply:
Link to l-loader PR: https://github.com/96boards-hikey/l-loader/pull/5

Reply:
Sorry, please hold. Some corrections required.

Reply:
Ok. All system go again. Thanks!